src/ralph/backends/lrs/base.py

```diff
@@ -53,3 +53,17 @@ def query_statements(self, params: StatementParameters) -> StatementQueryResult:
     @abstractmethod
     def query_statements_by_ids(self, ids: List[str]) -> list:
         """Returns the list of matching statement IDs from the database."""
+
+
+class BaseAsyncLRSBackend(BaseAsyncDataBackend):
+    """Base LRS backend interface."""
+
+    @abstractmethod
+    async def query_statements(
+        self, params: StatementParameters
+    ) -> StatementQueryResult:
+        """Returns the statements query payload using xAPI parameters."""
+
+    @abstractmethod
+    async def query_statements_by_ids(self, ids: List[str]) -> list:
+        """Returns the list of matching statement IDs from the database."""
```

Comment:
```suggestion
        """Yield statements with matching ids from the backend."""
```
Following your suggestion this is updated in the sync version too